.github/workflows/linux_wheel.yaml

```diff
@@ -34,11 +35,13 @@ jobs:
       with-xpu: disable
       with-rocm: disable
       with-cuda: disable
+      build-python-only: "disable"
```

Comment:
I learned about this option from @ahmadsharif1. I turned it on for both Linux and Mac, as I think it fits our use-case where we're also building C++ libraries.